lms/djangoapps/program_enrollments/api/v1/tests/test_views.py

```diff
@@ -1346,9 +1346,10 @@ def setUp(self):
         )
 
         # create course enrollment
-        self.course_enrollment = CourseEnrollmentFactory(
+        self.course_enrollment = CourseEnrollmentFactory.create(
```

Comment:
what does this change do?

Reply:
Makes it so that a record is created in the database.

Reply:
Oh, so just calling the factory creates the model but doesn't save it? Interesting!